### ISSUE TITLE
Make use of apt-add-repository in the instructions for the Debian package

### ISSUE THREAD

**Issue by schildbach** [matrix-org/synapse#5088]
**Description:**
Rather than [manually download and install a keyring](https://github.com/matrix-org/synapse/blob/master/INSTALL.md#debianubuntu), why not simply use `apt-add-repository` for the task? That would be the standard way to do it.

**Comment by richvdh** [comment]
I am not aware of a way to use `apt-add-repository` to install a key file?

**Comment by schildbach** [comment]
@richvdh It does this automatically if you use the "ppa:"-syntax. I'm a bit surprised, for the last 10 years or so I've only seen instructions adding repositories via `apt-add-repository`.

**Comment by richvdh** [comment]
the ppa syntax is specific to ubuntu's ppas. It doesn't work on debian, and would require us to manage a ppa as well as our regular repo. We don't have any plans to add more distribution mechanisms at the moment.

**Comment by schildbach** [comment]
It's hard to believe Debian has no user friendly mechanism for this.

I tried using [these Ubuntu instructions](https://www.natrius.eu/dokuwiki/doku.php?id=digital:server:matrixsynapse) recommended by matrix.org, but I now always get this error on `apt update`:

```
The following signatures couldn't be verified because the public key is not available: NO_PUBKEY F473DD4473365DE1
```
